commands/ps/scale.js

```diff
@@ -22,19 +22,41 @@ function * run (context, heroku) {
     }))
   }
 
-  let changes = parse(context.args)
-  if (changes.length === 0) {
-    let formation = yield heroku.get(`/apps/${app}/formation`)
+  const displayFormation = co.wrap(function * () {
+    const formation = yield heroku.get(`/apps/${app}/formation`)
     if (formation.length === 0) throw emptyFormationErr(app)
     cli.log(formation.map((d) => `${d.type}=${d.quantity}:${d.size}`).sort().join(' '))
-  } else {
-    yield cli.action('Scaling dynos', {success: false}, co(function * () {
-      let formation = yield heroku.request({method: 'PATCH', path: `/apps/${app}/formation`, body: {updates: changes}})
+  })
+
+  const autoscalingCheck = co.wrap(function * () {
+    let monitors
+    try {
+      const appID = (yield heroku.get(`/apps/${app}`)).id // TODO: metrics api will eventually not require app ID
+      monitors = yield heroku.get(`/apps/${appID}/formation/web/monitors`, {host: 'api.metrics.heroku.com'})
+    } catch (err) { // skip autoscaling check if any http errors
```

Comment:
One of the intentions here was to proactively *not* throw an error because this call is going to be made for *anyone who tries to scale their app* regardless of whether they have autoscaling set up. If the metrics api is down, then showing this warning is way less important than preventing someone from scaling. This is a convenience we offer users and it shouldn't get in the way of them performing basic platform actions